application/src/test/java/org/togetherjava/tjbot/commands/mediaonly/MediaOnlyChannelListenerTest.java

```diff
@@ -1,86 +1,62 @@
 package org.togetherjava.tjbot.commands.mediaonly;
 
-import net.dv8tion.jda.api.JDA;
-import net.dv8tion.jda.api.entities.EmbedType;
-import net.dv8tion.jda.api.entities.Message;
-import net.dv8tion.jda.api.entities.MessageEmbed;
-import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.entities.*;
 import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
-import net.dv8tion.jda.api.requests.RestAction;
-import net.dv8tion.jda.api.requests.restaction.AuditableRestAction;
-import net.dv8tion.jda.internal.JDAImpl;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.togetherjava.tjbot.config.Config;
 import org.togetherjava.tjbot.jda.JdaTester;
 
 import java.util.Collections;
-import java.util.List;
 
-import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.*;
 
 class MediaOnlyChannelListenerTest {
 
-    private JDA jda;
-    private final Message message = mock(Message.class);
-    private final long responseNumber = 1L;
-    private final User user = mock(User.class);
+    JdaTester jdaTester;
+    private Message message;
+    private User user;
     private MediaOnlyChannelListener mediaOnlyChannelListener;
 
     @BeforeEach
     void setUp() {
-        JdaTester jdaTester = new JdaTester();
-        jda = jdaTester.getJdaMock();
-
+        jdaTester = new JdaTester();
+        message = jdaTester.createMessageSpy();
+        Member memberSpy = jdaTester.createMemberSpy(1L);
+        user = memberSpy.getUser();
```

Comment:
if u want, u can just chain that call `user = jdaTester.getMemberSpy().getUser()`